src/test/java/core/basesyntax/servicesimpl/FruitShopSupplierImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.servicesimpl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import core.basesyntax.db.DataBase;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.services.FruitShopSupplier;
+import core.basesyntax.strategy.ElementDoesNotExist;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitShopSupplierImplTest {
+    private final FruitShopSupplier supplier = new FruitShopSupplierImpl();
+    private final List<String> listWithFruits = List.of("type,fruit,quantity",
+            "b,banana,20", "b,apple,100", "b,kivi,100", "s,banana,100",
+            "p,kivi,100", "p,banana,13", "r,apple,10", "p,apple,20", "p,banana,5",
+            "s,kivi,5", "s,kivi,20", "s,banana,50");
+    private List<String> listWithFruitsNull;
+    private Map<String, Fruit> rightMap;

```

Comment:
make it local and think about better naming

Reply:
✅.